frontend/probloom/src/components/Prob/Prob.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { shallow } from 'enzyme';
+
+import Prob from './Prob';
+
+describe('<Problem />', () => {
+  it('Clicked component should render without errors', () => {
+    const component = shallow(<Prob title='title' 
+        date='date' creator='creator' solved={0} recommended={0} clickProb={()=>{return}}/>);
```

Comment:
For your information, you can just use `() => {}` instead of `() => {return}`. You have to use `() => ({})` to make an arrow function that returns an object.